iOS/DuckDuckGo/AIChat/InputBox/SwitchBar/FlowManagers/DaxLogoManager.swift

```diff
@@ -106,6 +154,16 @@ private final class DaxLogoView: UIView {
 
     private struct Metrics {
         static let maxLogoSize: CGFloat = 96
+        static let maxDuckAILogoSize: CGFloat = 140
         static let spacing: CGFloat = 12
+        static let paddingDiff: CGFloat = maxDuckAILogoSize - maxLogoSize
+
+        static func maxLogoSize(isDuckAI: Bool) -> CGFloat {
+            isDuckAI ? maxDuckAILogoSize : maxLogoSize
+        }
+
+        static func spacing(isDuckAI: Bool) -> CGFloat {
+            isDuckAI ? spacing - (paddingDiff/2) : spacing
+        }
```

Comment:
Difference is caused by padding on DuckAI illustration. See https://app.asana.com/1/137249556945/task/1210814996510639/comment/1210902263043649?focus=true